WcaOnRails/app/models/team_member.rb

```diff
@@ -8,6 +8,7 @@ class TeamMember < ApplicationRecord
   scope :in_official_team, -> { where(team_id: Team.all_official.map(&:id)) }
   scope :leader, -> { where(team_leader: true) }
   scope :senior_member, -> { where(team_senior_member: true) }
+  scope :current_leader, -> { self.current.merge(self.leader) }
```

Comment:
Summarizing convo on slack:
This should be changed to sort the leaders and take the most recent leader. As well as change the naming to like single_current_leader.
This is due to there not being a restriction for a team having only 1 leader.